graylog2-web-interface/webpack.config.js

```diff
@@ -132,11 +130,6 @@ if (TARGET === 'build') {
   process.env.NODE_ENV = 'production';
   module.exports = merge(webpackConfig, {
     plugins: [
-      new webpack.DefinePlugin({
-        'process.env': {
-          NODE_ENV: JSON.stringify('production'),
-        },
-      }),
       new webpack.optimize.UglifyJsPlugin({
         minimize: true,
```

Comment:
We need to pass the `minimize` option into the loaders now, as apparently the `UglifyJsPlugin` does not do it any more: https://webpack.js.org/guides/migrating/#uglifyjsplugin-minimize-loaders